hypha/apply/users/forms.py

```diff
@@ -198,6 +199,15 @@ def clean_email(self):
             )  # updated email to avoid email existing message, fix information leak.
         return email
 
+    def clean_full_name(self):
+        full_name = self.cleaned_data["full_name"]
+        # Remove all HTML tags. This prohibits HTML without creating hurdles.
+        cleaned_full_name = nh3.clean(full_name, tags=set())
+        # Remove all colons and slashes to nerf URLs regardless of HTML tags.
+        cleaned_full_name = cleaned_full_name.replace(":", "")
+        cleaned_full_name = cleaned_full_name.replace("/", "")
+        return cleaned_full_name
```

Comment:
Looks good but can we break out this into a util function? Then we have this logic only in one place and can reuse it where needed.

Reply:
Good call. I left the function in the same file because this is its one use. If it needs to be moved elsewhere in future due to use outside this file or speculatively now that is also fine.